composables/use-likecoin-session-api.ts

```diff
@@ -275,6 +275,25 @@ export function useLikeCoinSessionAPI() {
     })
   }
 
+  async function sendCollectorMessage({
+    classId,
+    paymentId,
+    message,
+    wallet,
+    claimToken,
+  }: {
+    classId: string
+    paymentId: string
+    message: string
+    wallet: string
+    claimToken: string
+  }) {
+    return fetch.value<string>(`/likernft/book/purchase/class/${classId}/message/${paymentId}?token=${claimToken}`, {
+      method: 'POST',
+      body: { message, wallet },
+    })
```

Comment:
`ofetch` support query options
```suggestion
    return fetch.value<string>(`/likernft/book/purchase/class/${classId}/message/${paymentId}`, {
      method: 'POST',
      query: { token: claimToken },
      body: { message, wallet },
    })
```